src/content-script.ts

```diff
@@ -25,19 +26,26 @@ import { isBodyWithReplies, isRepositoryDetails } from "./validations";
 	}
 	const { default_branch: defaultBranch } = repositoryDetails;
 
-	// 3. Fetch the repository's .github/replies.yml
+	// 3. Fetch the REST API's JSON description of the item
```

Comment:
Speaking of documentation: do you have the time & energy to add some to the README.md? No worries if not, I can 🙂

Reply:
LGTM @JoshuaKGoldberg, thanks for digging in & finding the problem. I'm open to making some docs PRs. Did you want that done in this PR? And did you have a rough idea of what you wanted covered?

Reply:
> done in this PR?

I suppose it doesn't have to be done in this PR! I'll go ahead and merge it now to start the process of getting the update onto the app store.

> what you want covered

Maybe...
* The fact that you can use these values
* A quick example of them
* Links to the GitHub and Mustache docs

What do you think? I'll file a docs issue and repeat this there.